doc/releases/index.rst

```diff
@@ -1,14 +1,52 @@
 .. _zephyr_release_notes:
 
-Release Notes
-#############
+Releases
+########
 
 Zephyr project is provided as source code and build scripts for different
 target architectures and configurations, and not as a binary image. Updated
 versions of the Zephyr project are released approximately every three-months.
 
 All Zephyr project source code is maintained in a `GitHub repository`_.
 
+Release Life Cycle and Maintenance
+**********************************
+
+Periodic Releases
+=================
+
+The Zephyr project provides periodic releases every 4 months leading to the
```

Comment:
This says 4 months, but line 8 says "approximately every three-months"

Reply:
fixed, somewhere else we had two-months :(